OutOfSchool/OutOfSchool.AuthCommon/Controllers/AccountController.cs

```diff
@@ -465,6 +465,7 @@ public async Task<ResponseDto> LogOutUser(string userId)
         return await userService.LogOutUserById(userId);
     }
 
+    [FeatureGate(AuthServerConstants.FeatureManagement.EmailConfirmation)]
```

Comment:
technically private methods should not be disabled if the public method that uses them is disabled.